Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml

```diff
@@ -157,4 +165,4 @@ entityMappings:
     fieldMappings:
       - identifier: Address
         columnName: IPCustomEntity
-version: 1.2.0
\ No newline at end of file
+version: 1.0.0
```

Comment:
did you change it to 1.0.0 on purpose?
It might break customers that already use version 1.2.0.